src/reference/asciidoc/polling-consumer.adoc

```diff
@@ -59,6 +59,76 @@ _Version 4.2_ added more flexibility in this area - see <<conditional-pollers>>.
 This chapter is meant to only give a high-level overview regarding Polling Consumers and how they fit into the concept of message channels - <<channel>> and channel adapters - <<channel-adapter>>.
 For more in-depth information regarding Messaging Endpoints in general and Polling Consumers in particular, please see <<endpoint>>.
 
+[[deferred-acks-message-source]]
+==== Deferred Acknowledgment Pollable Message Source
+
+Starting with _version 5.0.1_, certain modules provide `MessageSource` implementations that support deferring acknowledgment until the downstream flow completes (or hands off the message to another thread).
+This is currently limited to the `AmqpMessageSource` and the `KafkaMessageSource` provided by the https://github.com/spring-projects/spring-integration-kafka[spring-kafka-integration extension project, version 3.0.1 or higher].
+
+With these message sources, the `ACKNOWLEDGMENT_CALLBACK` header (see <<message-header-accessor>>) is added to the message.
```

Comment:
`IntegrationMessageHeaderAccessor.ACKNOWLEDGMENT_CALLBACK` because the real header key is `acknowledgmentCallback`.